src/crimson/osd/osd_operations/internal_client_request.cc

```diff
@@ -53,88 +53,86 @@ CommonPGPipeline& InternalClientRequest::client_pp()
 seastar::future<> InternalClientRequest::start()
 {
   track_event<StartEvent>();
-  return crimson::common::handle_system_shutdown([this] {
```

Comment:
Why is it unnecessary? Looks if `system_shutdown_exception` is unhandled (i.e. thrown to the callers of `start()`), it will be leaked -- seems the returned futures are mostly ignored by the callers. CC @xxhdx1985126

Reply:
We call with_interruption immediately.  IOInterruptCondition includes both system_shutdown_exception and actingset_changed.  with_interruption should handle it in the interruption handler.

Reply:
Ah right, thanks!